pyproject.toml

```diff
@@ -14,6 +14,190 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+[build-system]
+requires = ["scikit-build-core>=0.10.0"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "tvm"
+version = "0.22.0.dev0"
```

Comment:
Add a comment that version.py needs to be called to update the version before we build the wheel, update 

https://github.com/apache/tvm/blob/main/version.py to replace the version here